pkg/install/v1alpha1/version_test.go

```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion(t *testing.T) {
+        tests := map[string]struct {
+                version    string
+                expversion string
```

Comment:
`expVersion` or `expectedVersion`

Reply:
@princerachit Just wanted to keep it short, so abbreviated expected to exp, you want me to change it?